config/tailwind.config.js

```diff
@@ -26,6 +26,7 @@ module.exports = {
         'gray-7': '#D7D7D8',
         'gray-6': '#C6C6C8',
         'gray-5': '#A2A2A2',
+        'gray-400': '#A2A2A2',
```

Comment:
Colors variables on Figma changed. We might need to update all the colors to have a 1:1 match.